src/webgl/p5.RendererGL.js

```diff
@@ -468,6 +473,17 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
     this.spotLightAngle = [];
     this.spotLightConc = [];
 
+    // Null if no image light is set, otherwise, it is set to a p5.Image
+    // made a blurrytexture attribute to be used in imageLight function
+    // this is to lookup image from blurrytexture Map rather from a
+    // texture map
+    this.diffusedTextures = new Map();
+    // map to store the created textures to prevent mis calculation
```

Comment:
This one maps the input `p5.Image` to a `p5.MipmapTexture`. Worth mentioning that we use a mipmap texture because it can contain light from multiple roughness levels.